Cron/Order.php

```diff
@@ -63,11 +69,14 @@ class Order
     /**
      * Default constructor
      *
+     * @param Config $config
      * @param Order $order
```

Comment:
I know this isn't your change, but we should probably take this opportunity to make the second `@param` in the docblock match what's actually getting passed to the constructor.

Reply:
done 👍

Reply:
Thanks! But the variable name needs to be updated too.